### ISSUE TITLE
Update ICOSCP metadata access

### ISSUE THREAD

**Issue by gareth-j** [openghg/openghg#381]
With newer ICOSCP the metadata access has been changed. This needs fixing as currently ICOS  data can't be retrieved.

```python
/site-packages/icoscp/cpb/dobj.py:160: FutureWarning: 
            ICOSCP release >= 0.1.15has changed 
            the format of .info and will return a dictionary containing
            all the metadata. The meta data should now be accessed through
            .meta
            
            This warning will be removed in future icoscp releases. Please
            update your code base accordingly. Documentation is available at
            https://icos-carbon-portal.github.io/pylib/modules/#dobj
            
  warn(msg, FutureWarning)
```

**Comment by gareth-j** [comment]
Tutorial 7 needs updating once this is fixed.

**Comment by gareth-j** [comment]
This should be fixed with #383
